docs/commands/alias.md

```diff
@@ -0,0 +1,96 @@
+<!-- DO NOT EDIT: this file is automatically generated using scw-doc-gen -->
+# Documentation for `scw alias`
+This namespace allows you to manage your aliases
+You can use multiple aliases in one command
+aliases in your commands are evaluated and you get completion
+  with: isl = instance server list
+    "scw isl <TAB>" will complete as "scw instance server list <TAB>"
+    "scw <TAB>" will complete "isl"
+
+  
+- [Create a new alias for a command](#create-a-new-alias-for-a-command)
+- [Delete an alias](#delete-an-alias)
+- [List aliases and their commands](#list-aliases-and-their-commands)
+
+  
+## Create a new alias for a command
+
+This command help you create aliases and save it to your config
+
+This command help you create aliases and save it to your config
```

Comment:
duplicate sentence

Reply:
Duplicated sentence is here because the `alias create` command is a command of resource level, the first one of its kind.
I create #2837